app/client/ui/DocumentSettings.ts

```diff
@@ -289,6 +303,102 @@ export class DocSettingsPage extends Disposable {
+    modal((ctl, owner) => {
+      this.onDispose(() => ctl.close());
+      const currentDocType = docPageModel.type.get() as string;
+      let currentDocTypeOption;
+      switch (currentDocType) {
+        case "":
+          currentDocTypeOption = DocTypeOption.Regular;
+          break;
+        case "Template":
+          currentDocTypeOption = DocTypeOption.Template;
+          break;
+        case "Tutorial":
+          currentDocTypeOption = DocTypeOption.Tutorial;
+          break;
+        default:
+          currentDocTypeOption = DocTypeOption.Regular;
+      }
+
+      const selected = Observable.create<DocTypeOption>(owner, currentDocTypeOption);
+
+      const doSetDocumentType = async () => {
+        const docId = docPageModel.currentDocId.get();
+        let docType;
+        if (selected.get() === DocTypeOption.Regular) {
+          docType = "";
+        } else if (selected.get() === DocTypeOption.Template) {
+          docType = "Template";
+        } else {
+          docType = "Tutorial";
+        }
+        persistType(docType, docId)
+        .then(()=>window.location.reload())
+        .catch(err=>console.log(err));
+
+        ctl.close();
+      };
+
+      const documentTypeOptions = () => [
+        cssRadioCheckboxOptions(
+          dom.style('max-width', '400px'),
+          radioCheckboxOption(selected, DocTypeOption.Regular, dom('div',
+            dom('div',
+              dom('strong', t('Regular document')),
+            ),
+            dom('div',
+              dom.style('margin-top', '8px'),
+              dom('span', t('Regular document behavior, all users work on the same copy of the document.'))
+            ),
+            testId('doctype-modal-option-regular'),
+          )),
+          radioCheckboxOption(selected, DocTypeOption.Template, dom('div',
+            dom('div',
+              dom('strong', t('Template')),
+            ),
+            dom('div',
+              dom.style('margin-top', '8px'),
+              dom('span',
+                t('Document automatically opens in {{fiddleModeDocUrl}}. ' +
+                  'Any edit (open to anybody) will create a new unsaved copy.',
+                  {
+                    fiddleModeDocUrl: cssLink({href: commonUrls.helpAPI, target: '_blank'}, t('fiddle mode'))
+                  }
+                )
+              ),
+            ),
+            testId('doctype-modal-option-template'),
+          )),
+          radioCheckboxOption(selected, DocTypeOption.Tutorial, dom('div',
+            dom('div',
+              dom('strong', t('Tutorial')),
+            ),
+            dom('div',
+              dom.style('margin-top', '8px'),
+              dom('span', t('Document automatically opens with a new copy.')),
+            ),
+            testId('doctype-modal-option-tutorial'),
+          ))
+        ),
+        cssModalButtons(
+          bigBasicButton(t('Cancel'), dom.on('click', () => ctl.close()), testId('doctype-modal-cancel')),
```

Comment:
Can't we make that a form, and use the standard buttons for these?

I feel like it would be help more complying with the accessibility standards.

Reply:
I'm not sure of what you mean by standard buttons as the BigBassicButtons are rendered by :

```
<button class="_grain12_ _grain12_-large test-settings-doctype-modal-cancel">Annuler</button>
<button class="_grain12_ _grain12_-large _grain12_-primary test-settings-doctype-modal-confirm">Confirm change</button>
```

which seems pretty standard to me.

I think you are right for the form. It means that other parts of the Document setting page needs to be adapted to be compliant with accessibility.

Reply:
I rather think of using [`<input type="submit" value="label for the button">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/submit) or [`<button type="submit">label for the button</button>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#type). The event you would have to listen to would be [`submit`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event) for the form.

Reply:
I'm interested to have a talk with @manuhabitela on this subject before make any changes. As other components in this page have the same issue, it's maybe a better idea to address it all at the same (at component level ?).

Reply:
:wave: 

Sorry I missed the notification of this :)

Sure in theory, using an actual form tag and listening to the submit event would _feel_ better I agree :) _Use the platform_ as they say. **But** there is no actual accessibility necessity to do it. There is no requirement for `form` tags in WCAG and I don't personally recall of any examples where users actually benefit of their presence.

Main things to check are input and button labels, keyboard navigation, error handling… but if everything behaves correctly thanks to specific HTML attributes and JS event listeners, no harm done :+1:

Reply:
I choose to let it consistent with the rest of the code on the setting page.
@fflorent if you still think that its important to fix it. I propose you to put it in another issue for the sake of delivering this one.